exec/cnex/cnex.c

```diff
@@ -47,15 +48,16 @@ void exec_freeExecutor(TExecutor *e)
         cell_clearCell(&e->globals[i]);
     }
     free(e->globals);
+    assert(isEmpty(e->stack));
     destroyStack(e->stack);
     free(e->stack);
     framestack_destroyFrameStack(e->framestack);
     free(e->framestack);
     free(e->module);
 
-    bytecode_freeBytecode(e->object);
+    //bytecode_freeBytecode(e->object);
 
-    free(e);
+    //free(e);
```

Comment:
Why is this commented out? If this function doesn't free the executor, then who does? And what should this function be named if it doesn't actually free the executor?

Reply:
It calls free on the TExecuter that is created, but the caller still has to free the actual pointer that was pointed to by it.  Is that not a good way to handle this?